ai_chatbots/models.py

```diff
@@ -53,6 +53,9 @@ class DjangoCheckpoint(models.Model):
     Checkpoint created by the DjangoSaver checkpointer class.
     """
 
+    created_on = models.DateTimeField(
```

Comment:
This is added directly instead of using TimestampedModel to allow null=True for old records